signals.go

```diff
@@ -41,17 +41,10 @@ func signalHandler(listener net.Listener, statusListener net.Listener, stopper c
 	for {
 		// Wait for a signal
 		select {
-		case sig := <-signals:
-			switch sig {
-			case syscall.SIGTERM:
-				logger.Printf("received SIGTERM, stopping listener")
+		case _ = <-signals:
```

Comment:
Who sends SIGTERM? We could add syscall.SIGTERM to signal.Notify. Letting the process die seems fine too?